pkg/validation/error.go

```diff
@@ -1,9 +1,76 @@
 package validation
 
 import (
+	"regexp"
+	"strings"
+
 	"github.com/giantswarm/microerror"
 )
 
+const (
+	appAdmissionControllerText string = "admission webhook \"apps.app-admission-controller-unique.giantswarm.io\" denied the request:"
+)
+
+var (
+	appConfigMapNotFoundPattern = regexp.MustCompile(`app config map not found error: configmap [\d\D]+ in namespace [\d\D]+ not found`)
+	kubeConfigNotFoundPattern   = regexp.MustCompile(`kube config not found error: kubeconfig secret [\d\D]+ in namespace [\d\D]+ not found`)
+)
+
+var appConfigMapNotFoundError = &microerror.Error{
+	Kind: "appConfigMapNotFoundError",
+}
+
+// IsAppConfigMapNotFound asserts appConfigMapNotFoundError.
```

Comment:
These error matchers are needed so cluster-operator can match the errors it gets from app-admission-controller.